examples/js/shaders/LUTShader.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @author miibond
+ *
+ * Full-screen tone-mapping shader based on http://www.cis.rit.edu/people/faculty/ferwerda/publications/sig02_paper.pdf
+ */
```

Comment:
I don't think this comment is correct. Same for the authorship.

Reply:
Doh 😅